MekHQ/src/mekhq/campaign/ResolveScenarioTracker.java

```diff
@@ -860,15 +858,11 @@ private void loadUnitsAndPilots(File unitFile) throws IOException {
             MULParser parser = new MULParser();
 
             // Open up the file.
-            InputStream listStream = new FileInputStream(unitFile);
-            // Read a Vector from the file.
-            try {
+            try (InputStream listStream = new FileInputStream(unitFile)) {
+                // Read a Vector from the file.
                 parser.parse(listStream);
-                listStream.close();
-            } catch (Exception excep) {
-                excep.printStackTrace(System.err);
-                // throw new IOException("Unable to read from: " +
-                // unitFile.getName());
```

Comment:
This is the only logical change, I've changed listStream into a try (with resources)